great_expectations/core/usage_statistics/schemas.py

```diff
@@ -365,6 +389,45 @@
     "additionalProperties": False,
 }
 
+test_yaml_config_payload_schema = {
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "test-yaml-config",
+    "definitions": {
+        "anonymized_string": anonymized_string_schema,
+        "anonymized_class_info": anonymized_class_info_schema,
+    },
+    "type": "object",
+    "properties": {
+        "anonymized_name": {"$ref": "#/definitions/anonymized_string"},
+        "parent_class": {"type": "string", "maxLength": 256},
+        "anonymized_class": {"$ref": "#/definitions/anonymized_string"},
+        "diagnostic_info": {
+            "type": "array",
+            "maxItems": 1000,
+            "items": {
+                "enum": [
+                    "__substitution_error__",
+                    "__yaml_parse_error__",
+                    "__custom_subclass_not_core_ge__",
+                    "__class_name_not_provided__",
```

Comment:
more descriptive errors: 
1. substitution : variable substitution error (I dont exist)
2. yaml parse : tabs are off in yaml
3. not core GE : the class is not in the list in anonymizer or datacontext
4. class_name not provided : we didn't have a class name for you